HW6_Function.py

```diff
@@ -0,0 +1,18 @@
+# Task 1
+def exponent_digits(x, y):
+    return x**y
```

Comment:
nit: use spaces before and after operator for more readability
```suggestion
    return x ** y
```